virttest/remote_commander/remote_runner.py

```diff
@@ -702,6 +712,49 @@ def exit(self):
         self._exit = True
         return "bye"
 
+    def run_test(self, test_path):
+        """
+        Call run() function at external python module.
```

Comment:
The remoter is a generic interface, so this name should reflect what it does and it could do way more than just running tests... I can't come up with a better name, but something like `python_file_call_run` or `python_file_run_with_helper` or ...